### DISCUSSION TITLE
libspdm and OpenBMC

### DISCUSSION THREAD

**Discussion by roberthbishop** [DMTF/libspdm#1885] [Q&A] [answered]
Sorry, I am new here, so I may be asking something that's already been answered, but is libspdm supported in OpenBMC?

If so, what mechanisms of SPDM messages are supported:
SPDM over MCTP over SMbus/I2C?
SPDM over MCTP over PCIe-VDM?
SPDM over PCIe-DOE?

Thx,
Bob

**Comment by steven-bellock** [comment, marked as answer]
That question is asked a lot but nobody seems to know the answer, so you may have better luck asking the OpenBMC folks.
